### PR TITLE
Add page for Ember CLI API Docs

### PR DESCRIPTION
**Pull request by mehulkar**
I had already done this before I saw the comment in #667. Figured i'd send it in. Seems like an improvement even if there is something better coming.

### CODE REVIEW

**Comment by sivakumar-kailasam** [comment]
Looks like the review app deployment failed. I'll have a look at this PR once locally & get this sorted out.

File: app/routes/ember-cli.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  titleToken() {
+    return 'Ember CLI'
+  }
+})
```
Comment (mehulkar) [inline app/routes/ember-cli.js]: @sivakumar-kailasam done. I didn't have time to look into how `titleToken` works, but with this change: `document.title` is "Ember - Ember CLI - Ember API Documentation". I tried empty string and not returning anything, but both of those remove the title entirely. In the other pages, that middle segment is the version, so there isnt' anything more appropriate possible here IMO. Let me know if you want a different string.

File: app/routes/ember-cli.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  titleToken() {
+    return 'Ember CLI';
+  }
```
Comment (mehulkar) [inline app/routes/ember-cli.js]: @sivakumar-kailasam done. I didn't have time to look into how `titleToken` works, but with this change: `document.title` is "Ember - Ember CLI - Ember API Documentation". I tried empty string and not returning anything, but both of those remove the title entirely. In the other pages, that middle segment is the version, so there isnt' anything more appropriate possible here IMO. Let me know if you want a different string.

Comment (sivakumar-kailasam) [reply app/routes/ember-cli.js]: I believe the `title` function should work here. Made that change in this PR. Will look at the review app once to be sure.

**Pull request merged**